Sources/DelegateProxy/ObjcDelegateProxy/ObjcDelegateProxy+Selector.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ObjcDelegateProxy+Selector.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 25/09/2019.
+//  Copyright © 2019 Joan Disho. All rights reserved.
+//
+
+import Foundation
+
+extension ObjcDelegateProxy {
+    @objc(canRespondToSelector:)
+    public func canRespond(to selector: Selector) -> Bool {
+        return selectors.map { $0 as? Selector }.contains(selector)
```

Comment:
```suggestion
        selectors.contains(where: { $0 as? Selector == selector })
```

Cheaper, and doesn't require going over all selectors